pandas/tests/frame/test_analytics.py

```diff
@@ -788,17 +803,14 @@ def alt(x):
         assert kurt.name is None
         assert kurt2.name == 'bar'
 
-    def _check_stat_op(self, name, alternative, frame=None, has_skipna=True,
+    # underscores added to distinguish argument names from fixture names
```

Comment:
can this just be a module level function now? maybe rename to assert_stat_ops

Reply:
I made this a module-level function, and broke it up into `assert_stat_op_calc` and `assert_stat_op_api`. This also removes the redundancy mentioned above.